src/md-collection-loader/description.js

```diff
@@ -0,0 +1,39 @@
+import remark from "remark"
+import stripMd from "strip-markdown"
+import { prune }  from "underscore.string"
+
+const defaultOpts = {
+  pruneLength: 140,
+  pruneString: "…",
+}
+
+export default function description(mdObject, opts = {}) {
+  opts = { ...defaultOpts, ...opts }
+
+  if (opts.pruneLength <= 0) {
+    throw new Error("Prune length must be larger than 0")
```

Comment:
Instead of throwing an error, I would prefer a warning more explicit like

```js
console.warning(
`You defined 'description.pruneLength' of md-collection-loader with an zero value. ` +
`This does not make sense, so the default value ${ defaultOpts.pruneLength } has been used. `
)
// or something like that.


// then use default value instead of 0.
// And maybe we can show this warning and use default when value is < 10 ?
```

I like having (and will try to do this more often) really verbose error/warning like React does.
I think it's really helpful.